libraries/dnsmadeeasy_record_cname.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'dnsmadeeasy_record'
+
+module DnsmadeeasyCookbook
+  class DnsmadeeasyRecordCname < DnsmadeeasyRecord
+    resource_name :dnsmadeeasy_record_CNAME
```

Comment:
Seems a little weird to have all caps for `CNAME`, might be nicer just as `dnsmadeeasy_record_cname`

Reply:
Yeah, I thought about it a lot — but in the DNS world they are always in caps. So I am just following DNS convention. Plus it helps to differentiate the type from the namespace.